### PR TITLE
Dx container site root

### PR DESCRIPTION
**Pull request by jaroel**
don't merge

### CODE REVIEW

**Comment by jensens** [comment]
@jaroel please dont use WIP in the title, but the labels provided. Using the labels I and others involved in PR management are able to filter the PRs and issues. Thanks!

File: CHANGES.rst [hunk 1]
```diff
@@ -179,6 +179,9 @@ New features:
 
 Bug fixes:
 
+- Fix ft._updateProperty so it doesn't break when receiving an empty value.
+  This happens when an DX FTI is part of a Generic Setup baseline import.
+  [jaroel]
```
Comment (jensens) [inline CHANGES.rst:184]: You need to put this log entry in a Towncrier conform news file.

Comment (jaroel) [reply CHANGES.rst]: Thanks! I've moved the entry to `news/85.bugfix`

File: plone/dexterity/schema.py [hunk 2]
```diff
@@ -326,7 +329,12 @@ def portalTypeToSchemaName(portal_type, schema=u"", prefix=None, suffix=None):
     """Return a canonical interface name for a generated schema interface.
     """
     if prefix is None:
-        prefix = '/'.join(getUtility(ISiteRoot).getPhysicalPath())[1:]
+        if portal_type == 'Plone Site':
+            fti = getUtility(IDexterityFTI, name=portal_type)
```
Comment (mauritsvanrees) [inline plone/dexterity/schema.py:333]: If we use `queryUtility` here, and catch the case where it is None, then this PR could probably be merged already before all the other dexterity site root PRs. It is a guess.

**Review by mauritsvanrees — APPROVED** [review]
LGTM.

I guess with a minor change (see inline comment) this might be mergeable already, even when the Plone Site is not yet dexterity.
I am going through the PRs to see which might be merged already, making the dexterity site root PLIP a bit smaller.

This would need yet another merge of master into this branch, or a rebase and force push, plus tested on its own in Jenkins.

File: plone/dexterity/schema.py (hunk 2 above)
Comment (mauritsvanrees) [reply plone/dexterity/schema.py]: Done.

**Comment by mauritsvanrees** [comment]
@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
Jenkins is green on 5.2 and 6.0 without needing the other site root PRs. For me it is fine to merge.

**Pull request merged**